docs/contribute.rst

```diff
@@ -208,14 +205,6 @@ few more at hand to further categorize issues.
     If the issue is urgent, assign this label. In the best case also go forward to
     resolve the ticket yourself as soon as possible.
 
-*Community Effort*
```

Comment:
We still have this label. :/ The other PR doesn't actually refer to removing it, it just removed it (unless I am missing something?)

Reply:
Well, here is a little of discussion https://github.com/rtfd/readthedocs.org/pull/4180#discussion_r193112104 and the commit https://github.com/rtfd/readthedocs.org/pull/4180/commits/d98189254c383c3f8de17ed0768581c8877df35f

Reply:
Yeah, I saw that discussion and the commit; it just doesn't seem like dropping the 'Community Effort' label was discussed, it was just included in the commit.

Reply:
I've already started retriaging these issues, but yes, there are still some left. I'm +1 on just removing from the docs for now.